### DISCUSSION TITLE
Input element inside modal receiving focus onshow

### DISCUSSION THREAD

**Discussion by batata004** [fomantic/Fomantic-UI#435] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Hi, the test case is here -> https://jsfiddle.net/8x4ncwth/

Whenever I open the modal using `modal("show")` the first input receives focus automatically. I have no way to disable this.

The way it works right now is really great on computers, it does not hurt to give focus automatically to the input field when the modal is being showed. BUT on smartphones this is terrible cause when the input receives focus it opens the keyboard interface of the device (on the screen) and the user cant read entirelly what is on the screen. The user has to close the keyboard and only after that the user can read what is fully on the screen.

Is there a way to disable this behaviour of automatically focusing the first input field just in smartphones?

**Comment by ColinFrick** [comment, marked as answer]
Use the `autofocus` setting: https://jsfiddle.net/rp23g7wn/

**autofocus** | true | When true, the first form input inside the modal will receive focus when shown.  Set this to false to prevent this behavior.

**Comment by batata004** [comment]
@ColinFrick thanks and sorry for posting this.
